pages/api/account/checkouts/renewal/checkouts.test.ts

```diff
@@ -123,24 +120,21 @@ describe("checkoutRenewal", () => {
       response: {
         status: 403,
         data: {
-          message:
+          description:
             "RENEWAL NOT ALLOWED. Please contact gethelp@nypl.org for assistance",
         },
       },
     })
     ;(sierraClient as jest.Mock).mockResolvedValueOnce({ post: clientMock })
 
-    try {
-      await checkoutRenewal(checkoutId)
-    } catch (error) {
-      expect(sierraClient).toHaveBeenCalled()
-      expect(clientMock).toHaveBeenCalledWith(
-        `patrons/checkouts/${checkoutId}/renewal`
-      )
-      expect(error.response.status).toBe(403)
-      expect(error.response.data.message).toBe(
-        "RENEWAL NOT ALLOWED. Please contact gethelp@nypl.org for assistance"
-      )
-    }
+    const response = await checkoutRenewal(checkoutId)
```

Comment:
Removing the try/catch and testing just what the function returns seems much nicer.